data/json/monsters/turrets.json

```
@@ -225,5 +225,55 @@
     "death_drops": {  },
     "death_function": [ "BROKEN" ],
     "flags": [ "SEES", "NOHEAD", "ELECTRONIC", "IMMOBILE", "NO_BREATHE", "DROPS_AMMO" ]
+  },
+  {
+    "id": "mon_turret_riot",
```

```suggestion
    "id": "mon_talon_m203",
```
Consistent monster IDs for UGVs. This should go in defense_bot.json too because we're likely to have moving ones at some point.